### DISCUSSION TITLE
Is it possible to run BAT files using Terminal?

### DISCUSSION THREAD

**Discussion by plonial** [microsoft/terminal#13013] [Q&A] [answered]
Is it possible to run BAT files using Terminal?

Translated by Google

**Comment by DHowett** [comment, marked as answer]
If you are using Windows 11, you can set Windows Terminal as your "default terminal application."

Reminder: batch files will continue to use Command Prompt, but they will appear inside Windows Terminal.

**Reply by plonial** [reply]
And Windows 10 has no solution?
Thank you

**Reply by DHowett** [reply]
Nope! Sorry.

**Reply by plonial** [reply]
😢

**Comment by thechampagne** [comment]
You can do it by write this line in cmd:
```bat
ftype batfile="%LOCALAPPDATA%\Microsoft\WindowsApps\wt.exe" "%1" %*
```
This will open batch files in terminal by default

**IF YOU WISH TO SET IT BACK**
write:
```bat
ftype batfile="%1" %*
```
